README.md

```diff
@@ -7,6 +7,17 @@ This repository contains two workflows for the analysis of Zika virus data:
 
 Each folder contains a README.md with more information.
 
+## Quickstart
+
+Follow the [standard installation instructions](https://docs.nextstrain.org/page/install.html) for Nextstrain's suite of software tools.
+
+Then run the default phylogenetic workflow via:
+```
+cd phylogenetic/
+nextstrain build .
+nextstrain view .
```

Comment:
This could be two lines as well:

```suggestion
nextstrain build phylogenetic/
nextstrain view phylogenetic/
```

Reply:
I just added a note on this here https://github.com/nextstrain/zika/pull/30/files#diff-0442ef409a305d27ae6b86f4cd48a1cf192e966cc8cf2e2da016fd63375408fbR17